Jenkinsfile

```diff
@@ -67,6 +67,12 @@ pipeline {
             }
         }
 
+        stage('Run pycodestyle and pylint3') {
+            steps {
+                sh './bin/run_pylint'
```

Comment:
Can we run pylint as part of docker build so that developer can identify the issues before raising PR ? 
Or after the docker build is complete, we need to start the container and get inside docker container and execute the command /bin/run_pylint ? Might be a good idea to add a docker compose file to run pylint.

Reply:
@manojgop Good idea--I will move invoking pylint to Docker, instead of delaying it to the Jenkins build.